imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```
@@ -0,0 +1,527 @@
+default_attrs: &default
+  # Assumed values for all variable attrs unless overwritten
+  DISPLAY_TYPE: time_series
+  FILLVAL: -9223372036854775808
+  FORMAT: I12
+  VALIDMIN: 0
+  VALIDMAX: 9223372036854775807
+  VAR_TYPE: data
+  UNITS: ' '
+
```

If you create this additional default, I think it will save lot of duplicates you have below. I added couple other suggestions using this additional default.
```suggestion
de_default_attrs: &de_default
  DISPLAY_TYPE: time_series
  FILLVAL: -9223372036854775808
  FORMAT: I12
  VALIDMIN: 0
  VALIDMAX: 9223372036854775807
  VAR_TYPE: support_data
  DEPEND_1: direct_events
  UNITS: ' '
```

---

Good idea. Added a default for this